### PR TITLE
DevWs for NIE-TECH CO., LTD. containing containing Z-Wave Switch Generic

### PR DESCRIPTION
**Pull request by sky-nie**

### CODE REVIEW

**Comment by KKlimczukS** [comment]
@sky-nie 
Since this branch has conflicts, could You resolve them first ?

File: devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy [hunk 1]
```diff
@@ -61,6 +61,8 @@ metadata {
 		fingerprint mfr: "0312", prod: "FF00", model: "FF03", deviceJoinName: "Minoston Switch" //Minoston Smart On/Off Switch
 		fingerprint mfr: "0312", prod: "C000", model: "CO05", deviceJoinName: "Evalogik Outlet", ocfDeviceType: "oic.d.smartplug" //Evalogik Mini Outdoor Smart Plug
 		fingerprint mfr: "031E", prod: "0004", model: "0001", deviceJoinName: "Inovelli Switch" //Inovelli Switch
+		fingerprint mfr: "001D", prod: "0037", model: "0002", deviceJoinName: "Leviton Outlet", ocfDeviceType: "oic.d.smartplug" //Leviton Tamper Resistant Outlet ZW15R
+        fingerprint mfr: "0312", prod: "FF00", model: "FF0C", deviceJoinName: "Minoston Mini Plug", ocfDeviceType: "oic.d.smartplug" //Minoston Mini Smart Plug
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy:65]: (After resolving conflicts on this branch)
Please add a Raw Description in a comment after Your fingeprint (You can find it in the SmartThings graph api in device details)  -> https://graph.api.smartthings.com/
so the fingerprint would look like -> 

fingerprint mfr: "0312", prod: "FF00", model: "FF0C", deviceJoinName: "Minoston Mini Plug", ocfDeviceType: "oic.d.smartplug" //Minoston Mini Smart Plug, Raw Description (paste raw description here)

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Comment by k-morawski** [comment]
Dear @sky-nie , @Winniewen18 , could you please check @KKlimczukS comments and reply? Please let us know if you need our help. Many thanks!

**Comment by sky-nie** [comment]
> Dear @sky-nie , @Winniewen18 , could you please check @KKlimczukS comments and reply? Please let us know if you need our help. Many thanks!

Sorry to trouble you, I updated this document today。

**Comment by KKlimczukS** [comment]
> > Dear @sky-nie , @Winniewen18 , could you please check @KKlimczukS comments and reply? Please let us know if you need our help. Many thanks!
> 
> Sorry to trouble you, I updated this document today。

@sky-nie , @Winniewen18
This branch still has conflicts. Please, could You resolve them first ?

**Comment by k-morawski** [comment]
Dear @sky-nie , @Winniewen18, could you please resolve your branch conflicts and make small fix - add a Raw Description in a comment after Your device's fingeprint ? @KKlimczukS gave you more info on that in his comment from Jan 13. 
If you fix it, we will merge your PR. Many thanks

**Pull request closed without merging**